packages/tools/devtools/devtools-browser-extension/e2e-tests/chromebrowserdevtool.test.ts

```diff
@@ -51,11 +51,11 @@ describe("End to end tests", () => {
 	});
 
 	// TODO
-	// it("Smoke: verify Devtools extension view can be launched", async () => {
-	// 	const targets = await browser.targets();
-	// 	console.log(targets);
-	// 	// chrome-extension://inmobceohkedafljagjfnbojplmlmgbk/devtools_app.html
-	// 	// document.querySelector("#containers-menu-section");
-	// 	// document.querySelector("#telemetry-menu-section");
-	// });
+	it("Smoke: verify Devtools extension view can be launched", async () => {
```

Comment:
Should we revert these changes? I don't think they're related to this PR's goals.